vispy/visuals/colorbar.py

```diff
@@ -679,5 +679,5 @@ def size(self, size):
         elif height == 0.:
             raise ValueError("height must be non-zero, not %s" % (height, ))
 
-        self._halfdim = (width, height)
+        self._halfdim = (width / 2, height / 2)
```

Comment:
Note the `.` at the end of the `2.` in my quote above -- it's in the code not as part of the sentence. If you do `x / 2.` the result will be a float in Python2 and Python3